### ISSUE TITLE
+ sign all over

### ISSUE THREAD

**Issue by WebReflection** [JairusSW/tutorial#3]
I am not sure if this is meant or anything, as it's literally everywhere in the documentation, but `+` sign at the beginning of each line is odd, somehow misleading, not something anyone should write/type, right?

**Comment by JairusSW** [comment]
Yeah. I believe its something that GitBook does. https://jtanaka.gitbook.io/guide/ 
I have it set to automatically push the source files to this GitHub repo.
Its in markdown format (with added features + syntax) so it must mean something to GitBook thats not valid MD. :)
